.rust_alpha/dynamodb/add-item/src/main.rs

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use std::process;
+
+use dynamodb::model::AttributeValue;
+use dynamodb::{Client, Config, Region};
+
+use aws_types::region::{ProvideRegion};
+
+use structopt::StructOpt;
+use tracing_subscriber::fmt::format::FmtSpan;
+use tracing_subscriber::fmt::SubscriberBuilder;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// The permission type of the user, standard_user or admin
```

Comment:
add period (to all comments on this page)

Reply:
Fixed